skimage/morphology/tests/test_flood_fill.py

```diff
@@ -40,7 +40,9 @@ def test_overrange_tolerance_float():
     image *= max_value
 
     expected = np.ones_like(image)
-    output = flood_fill(image, (0, 1), 1.0, tolerance=max_value * 10)
+    with np.errstate(over="ignore"):
+        tolerance = max_value * 10
```

Comment:
Hrm, is this safe?

Reply:
Good catch, I used the wrong approach to "fix" this as previously the result was using Python scalars while now it results in an `np.float32(inf)` which is not the same as before. I'll address this in a follow-up PR. :+1: